src/main/resources/arrebol.json

```diff
@@ -1,7 +1,7 @@
 {
   "poolType":"docker",
   "workerPoolSize":"5",
-  "imageId":"wesleymonte/simple-worker:latest",
+  "imageId":"wesleymonte/simple-worker",
```

Comment:
I think that is good to have a sanity test to check whether the user is specifying the image tag. If the image tag is specified, we should not start the application.

Reply:
This was solved in 2e9e19eabfbe623709bf2973a45531473bb93671.